pkg/util/pod.go

```diff
@@ -243,3 +243,7 @@ func GetPodRequest(pod *corev1.Pod, resourceNames ...corev1.ResourceName) corev1
 	}
 	return result
 }
+
+func TerminatedPod(pod *corev1.Pod) bool {
```

Comment:
this function name is a little bit confusing, I would assume it terminates a pod.

Reply:
use `IsPodTerminated`.